siriuspy/siriuspy/clientconfigdb/types/global_config.py

```diff
@@ -1194,63 +1197,64 @@ def get_dict():
+    ['BO-09U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-11U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-13U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-15U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-17U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-19U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-21U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-23U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-25U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-27U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-29U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-31U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-33U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-35U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-37U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-39U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-41U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-43U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-45U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-47U:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-49D:PS-CH:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-01U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-03U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-05U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-07U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-09U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-11U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-13U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-15U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-17U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-19U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-21U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-23U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-25U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-27U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-29U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-31U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-33U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-35U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-37U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-39U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-41U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-43U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-45U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-47U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
+    ['BO-49U:PS-CV:Wfm-SP', _np.array(_DEF_WFMSIZE_FBP), 0.0],
```

Comment:
The constants DEF_WFMSIZE_OTHERS/DEF_WFMSIZE_FBP and DEFAULT_WFMDATA have different meaning. To keep the same meaning, it is necessary to create arrays with these sizes.